modules/aws/s3.go

```diff
@@ -37,6 +37,9 @@ func FindS3BucketWithTagE(t testing.TestingT, awsRegion string, key string, valu
 		tagResponse, err := s3Client.GetBucketTagging(&s3.GetBucketTaggingInput{Bucket: bucket.Name})
 
 		if err != nil {
+			if strings.Contains(err.Error(), "NoSuchBucket") {
+				continue
+			}
```

Comment:
NIT: please add as a comment the description from the PR explaining why this error is safe to continue.

Reply:
Comment added